scripts/checkpatch_inc.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-CHECKPATCH=checkpatch.pl
+CHECKPATCH="${CHECKPATCH:-checkpatch.pl}"
```

Comment:
That will still work, I just like being able to easily change tools from the command line if I need, without messing with my PATH.